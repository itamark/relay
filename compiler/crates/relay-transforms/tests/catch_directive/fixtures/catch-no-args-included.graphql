fragment EnabledFoo on User {
  name @catch
}
