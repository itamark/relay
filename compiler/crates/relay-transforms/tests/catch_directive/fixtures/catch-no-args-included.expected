==================================== INPUT ====================================
fragment EnabledFoo on User {
  name @catch
}
==================================== OUTPUT ===================================
fragment EnabledFoo on User {
  name @catch @__CatchMetadataDirective
  # CatchMetadataDirective {
  #     to: Result,
  #     path: "name",
  # }
  
}
